src/game/option_passport.c

```diff
@@ -21,23 +21,22 @@ typedef enum PASSPORT_PAGE {
     PASSPORT_PAGE_COUNT = 3,
 } PASSPORT_PAGE;
 
-typedef enum PASSPORT_MODE {
-    PASSPORT_MODE_FLIP = 0,
-    PASSPORT_MODE_SHOW_SAVES = 1,
-    PASSPORT_MODE_NEW_GAME = 2,
-} PASSPORT_MODE;
-
 typedef enum PASSPORT_TEXT {
     TEXT_PAGE_NAME = 0,
     TEXT_LEFT_ARROW = 1,
     TEXT_RIGHT_ARROW = 2,
-    TEXT_NUMBER_OF = 3,
+    TEXT_LEVEL_ARROW_RIGHT = 3,
+    TEXT_LEVEL_ARROW_LEFT = 4,
+    TEXT_NUMBER_OF = 5,
 } PASSPORT_TEXT;
 
 static PASSPORT_MODE m_PassportMode = PASSPORT_MODE_FLIP;
 static bool m_IsTextInit = false;
 static TEXTSTRING *m_Text[TEXT_NUMBER_OF] = { 0 };
 static char m_NewGameStrings[MAX_GAME_MODES][MAX_GAME_MODE_LENGTH] = { 0 };
+static char m_LoadSavegameStrings[MAX_SAVE_SLOTS][MAX_LEVEL_NAME_LENGTH] = {
```

Comment:
Select level is limited to 15 levels (ignores gym level 1). This matches the PS1's 16 slots because we have save anywhere in the Load Game menu which is basically Current Position. This would limit custom levels to 15 levels + 1 gym too. If we want to dynamically read the levels from the gameflow, this would have to change from a static. What are your thoughts?

Reply:
We should compute this from the gameflow.

Something like (untested pseudocode)
```c
int32_t GameFlow_GetPlayableLevelCount(void)
{
    int32_t level_count = 0;
    for (int i = 0; i < g_GameFlow.level_count; i++) {
        if (g_GameFlow.levels[i].type == GFL_NORMAL) {
            level_count++;
        }
    }
    return level_count;
}
```

Reply:
Apparently this is already calculated and stored in `g_GameFlow.last_level_num`. Your code did work except for one wrong name. Sorry!